api/common/src/test/java/io/opentelemetry/api/common/AttributesTest.java

```diff
@@ -118,10 +118,7 @@ void asMap() {
     assertThat(map.values().contains("value1")).isTrue();
     assertThat(map.values().contains("value3")).isFalse();
 
-    assertThat(map.toString())
-        .isEqualTo(
-            "ReadOnlyArrayMap{AttributeKeyImpl{getType=STRING, key=key1}=value1,"
-                + "AttributeKeyImpl{getType=LONG, key=key2}=333}");
+    assertThat(map.toString()).isEqualTo("ReadOnlyArrayMap{key1=value1,key2=333}");
```

Comment:
shouldn't this one have quotes around the `value1` ?

Reply:
This is `Map`, not `Attributes`